packages/patterns/ct-checkbox-simple.tsx

```diff
@@ -0,0 +1,61 @@
+/// <cts-enable />
+import {
+  Cell,
+  Default,
+  h,
+  handler,
+  ifElse,
+  NAME,
+  recipe,
+  UI,
+} from "commontools";
+
+interface CheckboxSimpleInput {
+  enabled: Default<boolean, false>;
+}
+
+interface CheckboxSimpleOutput extends CheckboxSimpleInput {}
+
+export default recipe<CheckboxSimpleInput, CheckboxSimpleOutput>(
+  "ct-checkbox simple demo",
+  ({ enabled }) => {
+    // Handler for checkbox changes
+    const toggle = handler<
+      { detail: { checked: boolean } },
+      { enabled: Cell<boolean> }
+    >(
+      ({ detail }, { enabled }) => {
+        enabled.set(detail?.checked ?? false);
+      },
+    );
+
+    const toggleHandler = toggle({ enabled });
+
+    return {
+      [NAME]: "Checkbox Demo",
+      [UI]: (
+        <common-vstack gap="md" style="padding: 2rem; max-width: 400px;">
+          <h3>Simple ct-checkbox + ifElse Demo</h3>
+
+          <ct-checkbox
+            $checked={enabled}
```

Comment:
oh, the $ in $checked would imply to me that we don't need the event handler to change the state?

Reply:
I had intended this as a demo of there being two ways, but on reflection I think I should split the recipe into two.